omnichain/swap/tasks/deploy.ts

```diff
@@ -5,11 +5,11 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
 const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
   const network = hre.network.name as ParamChainName;
 
-  if (!/zeta_(testnet|mainnet)/.test(network)) {
-    throw new Error(
-      '🚨 Please use either "zeta_testnet" or "zeta_mainnet" network to deploy to ZetaChain.'
-    );
-  }
+  // if (!/zeta_(testnet|mainnet)/.test(network)) {
```

Comment:
if it's no longer valid just delete it, let's try to keep code clean

Reply:
Reverted, because it's still relevant: 4095527